manim/utils/testing/_frames_testers.py

```diff
@@ -38,7 +40,7 @@ def testing(self):
                 f"when there are {self._number_frames} control frames for this test."
             )
 
-    def check_frame(self, frame_number: int, frame: np.ndarray):
+    def check_frame(self, frame_number: int, frame: np.ndarray) -> None:
```

Comment:
Since the result of a renderer's `.get_frame()` method gets passed as `frame`, a better annotation for it would be `PixelArray`.

```suggestion
    def check_frame(self, frame_number: int, frame: PixelArray) -> None:
```